src/views/List.jsx

```diff
@@ -13,9 +24,11 @@ export function List({ data, listToken }) {
 		setSearchTerm('');
 	};
 
-	const filteredData = data.filter((item) =>
-		item.name.toLowerCase().includes(searchTerm.toLowerCase()),
-	);
+	const filteredData = sortedData.map((category) => {
+		return category.filter((item) => {
+			return item.name.toLowerCase().includes(searchTerm.toLowerCase());
+		});
+	});
```

Comment:
This should probably be wrapped in a `useMemo` with a dependency on `searchTerm`